tests/backup/backup_helper.go

```diff
@@ -2629,6 +2630,21 @@ func AreSlicesEqual(slice1, slice2 interface{}) bool {
 	return true
 }
 
+// AreStringSlicesEqual compares two slices of string
+func AreStringSlicesEqual(slice1 []string, slice2 []string) bool {
```

Comment:
I think AreSlicesEqual in backup_helper will work for strings as well.

Reply:
I'll check

Reply:
AreStringSlicesEqual function has been used in many places where as AreSlicesEqual not being used anywhere in torpedo. Hence I would suggest to go for AreStringSlicesEqual

Reply:
Make sense.